dist/angular-editable-text.js

```diff
@@ -26,13 +26,12 @@
         onChange: '&ggOnChange'
       },
       transclude: true,
```

Comment:
I was thinking about this last night and wondering if I should add the div instead of replacing these lines.  I don't think it will affect the functionality negatively, once I thought about it, and that way we wouldn't be losing functionality. I'll try it today.